src/input/scroll.c

```diff
@@ -6,8 +6,10 @@
 #include "core/time.h"
 #include "game/settings.h"
 #include "graphics/screen.h"
+#include "platform/screen.h"
 
 #include <math.h>
+#include <SDL_mouse.h>
```

Comment:
Don't use `SDL` related stuff outside of `platform` please. If you need, add a new function in `platform`.

Reply:
Looks like we don't have a platform file for the mouse. Would you recommend me to create a new one?